nessie/src/main/java/org/apache/iceberg/nessie/NessieUtil.java

```diff
@@ -102,6 +102,38 @@ private static String commitAuthor(Map<String, String> catalogOptions) {
         .orElseGet(() -> System.getProperty("user.name"));
   }
 
+  private static void checkAndUpdateGCProperties(
+      TableMetadata tableMetadata, Map<String, String> updatedProperties, String identifier) {
+    if (tableMetadata.propertyAsBoolean(
```

Comment:
Just thinking: should we add the lines
```java
updatedProperties.putIfAbsent(TableProperties.GC_ENABLED, "false");
updatedProperties.putIfAbsent(TableProperties.METADATA_DELETE_AFTER_COMMIT_ENABLED, "false");
```
here at the beginning of the function, to ensure these values do always default to `false` for Nessie?

Reply:
I think "table default" values from the catalog are enough. They will be set in new tables by Iceberg core.

Old tables have the forced `false` values from the older version of this this class anyway.